.github/workflows/install-from-condalock-file.yml

```
@@ -0,0 +1,103 @@
+# Install esmvaltool from a conda lock file
+# To build a conda lock file install conda-lock first then
+# run conda-lock lock --platform linux-64 -f environment.yml --mamba
+# (mamba activated for speed). Change platform for osx-64 or win-64.
+# Env creation then happens as per normal use with
+# conda create --name esmvaltool-fromlock --file conda-linux-64.lock
+# note that pip and conda are NOT installed.
+
+name: Conda-lock Install
+on:
+  push:
+    branches:
+    - main
```

This still makes sense to me, but fix indentation?
```suggestion
on:
  push:
    branches:
      - main
```